readthedocs/doc_builder/backends/mkdocs.py

```diff
@@ -112,9 +110,6 @@ def append_conf(self, **__):
         :raises: ``MkDocsYAMLParseError`` if failed due to known type errors
                  (i.e. expecting a list and a string is found).
         """
-        if not self.yaml_file:
-            self.yaml_file = os.path.join(self.root_path, 'mkdocs.yml')
-
```

Comment:
This is already handled below

And we check for not existing files here


https://github.com/rtfd/readthedocs.org/blob/7c661a346a75df151d9852d016242fa50bbf63fc/readthedocs/doc_builder/backends/mkdocs.py#L87-L92